helpers/channels/forums/mod.ts

```diff
@@ -1 +1 @@
-export * from "./createForumPost.ts";
+export * from "./startForumThread.ts";
```

Comment:
thoughts on naming this createThreadInForum or createForumThread

Reply:
`createForumThread` sounds good. Will update.